packages/ag-charts-enterprise/src/axes/angle/angleAxis.ts

```diff
@@ -194,7 +194,7 @@ export abstract class AngleAxis<
         const { label, tickLabelGroupSelection } = this;
 
         const ticks = this.tickData;
-        tickLabelGroupSelection.update(label.enabled ? ticks : []).each((node, _, index) => {
+        tickLabelGroupSelection.update(label.enabled ? (ticks as any[]) : []).each((node, _, index) => {
```

Comment:
👀 any?

Reply:
Will see if I can address in the follow-up PR I'll send shortly.